src/components/Marketplace/Checkout/useLongPress.js

```diff
@@ -0,0 +1,33 @@
+import { useState, useEffect, useCallback } from 'react';
+
+export default function useLongPress(callback = () => {}, ms = 300) {
```

Comment:
This component is super clever and concise, but the bug you mentioned definitely appears :(